src/components/modals/dialog.test.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Dialog from './Dialog'
+
+describe('Dialog component', () => {
```

Comment:
I wonder if instead of just snapshot testing, we should test some of the important behavior like if `onDismiss` will in fact be called or the scrolling.

Reply:
True.

Me and Adam are going to possibly talk about a more robust testing strategy next week potentially (I really don't like snapshots) but in the meantime, I can add those tests.

Reply:
:whynotboth: 🙃

Reply:
I don't dislike many things, but snapshot tests are one of them 🙂That being said, I've added both.